### PR TITLE
Updating Test Packages

### PR DESCRIPTION
**Pull request by votemike**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,13 +26,14 @@
...
+        "phpunit/phpunit": "^6.1",
+        "squizlabs/php_codesniffer": "^2.9",
         "symfony/cache": "^3.2"
     },
     "config": {
         "platform": {
             "php": "7.1.0"
-        }
+        },
+        "sort-packages": true
```
Comment (BPScott) [inline composer.json]: 😍

File: tests/Mapper/ProgrammesDbToDomain/MasterBrandMapperTest.php [hunk 2]
```diff
@@ -7,7 +7,7 @@
 use BBC\ProgrammesPagesService\Domain\Entity\Unfetched\UnfetchedVersion;
 use BBC\ProgrammesPagesService\Domain\Entity\Unfetched\UnfetchedNetwork;
 use BBC\ProgrammesPagesService\Domain\ValueObject\Mid;
-use PHPUnit_Framework_TestCase;
+use PHPUnit\Framework\TestCase;
```
Comment (BPScott) [inline tests/Mapper/ProgrammesDbToDomain/MasterBrandMapperTest.php]: unused?

File: tests/Mapper/ProgrammesDbToDomain/NetworkMapperTest.php [hunk 3]
```diff
@@ -6,7 +6,7 @@
 use BBC\ProgrammesPagesService\Mapper\ProgrammesDbToDomain\NetworkMapper;
 use BBC\ProgrammesPagesService\Domain\Entity\Network;
 use BBC\ProgrammesPagesService\Domain\ValueObject\Nid;
-use PHPUnit_Framework_TestCase;
+use PHPUnit\Framework\TestCase;
```
Comment (BPScott) [inline tests/Mapper/ProgrammesDbToDomain/NetworkMapperTest.php]: unused?

File: tests/Mapper/ProgrammesDbToDomain/ServiceMapperTest.php [hunk 4]
```diff
@@ -9,7 +9,7 @@
 use BBC\ProgrammesPagesService\Domain\ValueObject\Pid;
 use DateTime;
 use DateTimeImmutable;
-use PHPUnit_Framework_TestCase;
+use PHPUnit\Framework\TestCase;
```
Comment (BPScott) [inline tests/Mapper/ProgrammesDbToDomain/ServiceMapperTest.php]: unused?

File: composer.json (hunk 1 above)
Comment (votemike) [inline composer.json]: ❤️ Order

**Pull request merged**